ledger/notary-plugin/src/main/java/net/corda/v5/ledger/notary/pluggable/package-info.java

```diff
@@ -0,0 +1,4 @@
+@Export
+package net.corda.v5.ledger.notary.pluggable;
```

Comment:
```suggestion
package net.corda.v5.ledger.notary.plugin;
```

Consistency with module name; notaries are pluggable by default in C5, so I assume what we're really saying is that this module / package is providing the plugin core / infrastructure? With that in mind I'd also be open to the module / package referring to `plugin-core` or the like if we thought that made sense.

(Comment applies to all files in this package)

Reply:
All these interfaces in this package are needed to *create a plugin*, not related to an actual plugin. I'd argue it's better to be in `pluggable` since they are required to create a plugin thus "pluggable". 

A `plugin-core` module name would make sense to me.

That's just my opinion though, if you still want to stick with `plugin` I can modify it.

Reply:
Discussed this offline, will stick to having `plugin` as the root package and `api`, `core` as sub-packages.